app/github_oauth.py

```diff
@@ -39,6 +41,7 @@ def generate_github_access_token(github_code):
         r"access_token=([a-zA-Z0-9]+)", github_response.content.decode("utf-8")
     )
     if not token:
+        logging.warning("Wrong access token.")
```

Comment:
Can be rephrased as "Github access token is invalid" and log level should be error